### DISCUSSION TITLE
not getting approved for github student developer pack

### DISCUSSION THREAD

**Discussion by Kesaab** [community/community#27790] [Other Feature Feedback, Questions, & Ideas]
hi..i am keshab ,,i am a student of mca from north odisha university,
i had tried so many times to get the pack but evert time i am getting rejected ..
can someone do something..??

**Comment by FahdSeddik** [comment]
Hey! @Kesaab 
I hope you find an answer [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/why-wasnt-my-application-for-a-student-developer-pack-approved).

**Comment by mpboom** [comment]
@FahdSeddik thanks for sharing that documentation, that is indeed right. I'd like to add that on the forum we can't discuss these account-related matters.
